### PR TITLE
Exclude tests from Docker + fix path + bump version

### PR DESCRIPTION
**Pull request by tovbinm**
## Description

Exclude tests from Docker + fix path + bump version

## Type of change
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

### CODE REVIEW

**Comment by tovbinm** [comment]
@Roma-Kyrnis I am fixing it right now in this PR - https://github.com/faros-ai/airbyte-connectors/pull/101

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -73,14 +73,15 @@ jobs:
...
-          sudo chown -R $USER:$USER $(find ${{ github.workspace }}/sources/jenkins-source/test_files/data/.java/fonts -type f)
+          sudo chown -R $USER:$USER $(find ${{ github.workspace }}/sources/jenkins-source/test_files/jenkins/data/.java/fonts -type f)
           for i in $(ls -d sources/*/)
           do
             ./scripts/source-acceptance-test.sh $(echo $i | cut -f2 -d'/')
           done
         env:
           EXAMPLE_SOURCE_TEST_CREDS: ${{ secrets.EXAMPLE_SOURCE_TEST_CREDS }}
           JENKINS_SOURCE_TEST_CREDS: ${{ secrets.JENKINS_SOURCE_TEST_CREDS }}
+          PHABRICATOR_SOURCE_TEST_CREDS: ${{ secrets.PHABRICATOR_SOURCE_TEST_CREDS }}
```
Comment (cjwooo) [inline .github/workflows/ci.yml:84]: These and the corresponding Github secrets need to be renamed to `..._CONFIG`.

**Review by cjwooo — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cjwooo) [reply .github/workflows/ci.yml:84]: Actually these should just be removed.

Comment (cjwooo) [reply .github/workflows/ci.yml]: I've updated the CI workflow.

**Review by cjwooo — APPROVED** [review]

**Pull request merged**